test/command/6925.md

```diff
@@ -23,12 +23,12 @@ a
 ::: thm
 **Theorem 1**. *a*
 
-![image](1.png)
+![](1.png)
```

Comment:
Here the alt text disappears -- won't that mean we no longer have a figure?  An `implicit_figure` needs a non-empty alt text.

Reply:
Fixed.;